server/repository/src/migrations/v1_01_11/program_requisition.rs

```diff
@@ -0,0 +1,44 @@
+use crate::{
+    migrations::{sql, DOUBLE},
+    StorageConnection,
+};
+
+pub(crate) fn migrate(connection: &StorageConnection) -> anyhow::Result<()> {
+    sql!(
+        connection,
+        r#"
+            CREATE TABLE program (
+                id TEXT NOT NULL REFERENCES master_list(id) PRIMARY KEY,
+                name TEXT NOT NULL
```

Comment:
I wonder if we should enforce a unique name on programs?

Reply:
I also wonder if we should have an `id` and `master_list_id` (which would have the reference on it)?
Just incase we change the model in the future? Would be more consistent?

Reply:
program_ID in central refers to the master_list ID, so I thought it would be alright to do it this way (easier to test and translate...)

Store preference table also does something similar and uses the store_id as an id..

Reply:
Ok cool, good to stick to the existing pattern.
I just thought having a separate field might be more intuitive to someone looking at it later on.

Reply:
my review is taking a while, and you've commented after my initial comment Roxy!
fine with either - I liked JB's suggestion, but happy to leave it.